src/client/components/robot_selector/robot_label/view_model.tsx

```diff
@@ -0,0 +1,22 @@
+import { observable } from 'mobx'
+import { computed } from 'mobx'
+import { createTransformer } from 'mobx-utils'
+
+import { RobotNetworkStatsModel } from '../../../network/robot_network_stats_model'
+import { RobotModel } from '../../robot/model'
+
+export class RobotLabelViewModel {
+  @observable statsOpen = false
```

Comment:
View models shouldn't have any state, this should go on a plain model somewhere..